mvc/src/main/java/webmvc/org/springframework/util/ReflectionUtils.java

```diff
@@ -1,14 +1,24 @@
 package webmvc.org.springframework.util;
 
 import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Modifier;
 
 /**
  * Class like core.org.springframework.util.ReflectionUtils
  */
 public class ReflectionUtils {
 
-    public static <T> Constructor<T> accessibleConstructor(Class<T> clazz, Class<?>... parameterTypes)
+    public static <T> Object instantiate(Class<T> clazz) {
```

Comment:
```suggestion
    public static <T> T instantiate(Class<T> clazz) {
```
처럼 인스턴스화 되는 클래스로 전달하는 방법은 어떻게 생각하시나요?

Reply:
오 이게 더 좋은 방식이네요! 말씀처럼 수정하겠습니다 감사합니다~